src/seedu/addressbook/ui/Formatter.java

```diff
@@ -24,6 +26,17 @@
     /** Offset required to convert between 1-indexing and 0-indexing.  */
     private static final int DISPLAYED_INDEX_OFFSET = 1;
 
+    /** Delimiter used for a a list of printables */
+    private static final String DELIMITER = ", ";
+
+    /** Returns a concatenated and delimited string of printables */
+    public static String getPrintableString(Printable... printables) {
```

Comment:
What would be a better place to have this method? Formatter has a role to play, to format. Does this method fit in here without violating SRP?

Reply:
The only other place I can think of is common/Utils.